main/webapp/modules/core/langs/translation-en.json

```diff
@@ -680,6 +681,18 @@
     "core-views/use-values-as-identifiers2": "Mark cells as reconciled with their values as identifiers",
     "core-views/choose-reconciliation-service": "Choose a reconciliation service",
     "core-views/choose-reconciliation-service-alert": "Please choose a reconciliation service first.",
+    "core-views/column-join": "Join columns",
+    "core-views/column-join-before-column-picker": "Select and order columns to join",
+    "core-views/column-join-before-options": "Select options",
+    "core-views/column-join-field-separator": "Separator between the content of each column :",
+    "core-views/column-join-field-separator-advice": "Enter one or more character, or keep blank to juxtapose the columns.",
+    "core-views/column-join-replace-nulls": "Replace nulls with...",
+    "core-views/column-join-skip-nulls": "Skip nulls.",
+    "core-views/column-join-replace-nulls-advice": "Enter one or more character, or keep blank to replace nulls with blank strings.",
+    "core-views/column-join-skip-nulls-advice": "Skipping nulls will make impossible the reverse the join.",
```

Comment:
grammar improvement, "Skipping nulls will make it impossible to reverse the join."
But I am confused why we would mention this, since we have Undo/Redo for users to use?

Reply:
You are right! What I wanted was to describe the effects of join (["a",null,null,"b"],";") --> "a;b" in a few words, and I am not sure that "Skip nulls" is clear enough...
I believe that most use cases require the "coalesce" option, to keep trace of nulls, but in some cases it could be usefull to skip nulls.
Any idea of a better sentence? If not I will remove it, as it is purely informative.

Reply:
I would remove it.  The effects of join will show quite clearly, and if they user didn't like what they saw, then they simply undo.